### PR TITLE
Feature/Add Search to Navbar [EOSF-26]

### PR DESCRIPTION
**Pull request by pattisdr**
## Ticket
https://openscience.atlassian.net/browse/EOSF-26

# Purpose

Adds search to navbar.  Gets rid of unnecessary imports.

# Notes for Reviewers

### CODE REVIEW

File: addon/components/search-dropdown/template.hbs [hunk 1]
```diff
@@ -1,16 +1,15 @@
-<div class="osf-search" data-bind="fadeVisible: showSearch" style="display: none">
+<div class="osf-search">
     <div class="container">
         <div class="row">
             <div class="col-md-12">
-                <form class="input-group" data-bind="submit: submit">
-                    <input id="searchPageFullBar" name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search" data-bind="value: query, hasFocus: true">
+                <form class="input-group" {{action "search" on="submit"}}>
+                    <input id="searchPageFullBar" onchange={{action "updateQuery"}} name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search">
```
Comment (samchrisinger) [inline addon/components/search-dropdown/template.hbs]: You can use `value="target.value` here, and then update the signature of `updateQuery` to just take a value

**Comment by samchrisinger** [comment]
One tiny comment, otherwise lgtm

File: addon/components/search-dropdown/template.hbs (hunk 1 above)
Comment (pattisdr) [inline addon/components/search-dropdown/template.hbs]: I tried that originally, but then the placeholder was target.value, which I didn't want.

**Comment by samchrisinger** [comment]
@pattisdr looks like the search bar might be getting hidden for small screens. Can you please take a look at this?

**Comment by pattisdr** [comment]
Yes - thanks, I didn't check small screens.

File: addon/components/search-dropdown/template.hbs (hunk 1 above)
Comment (samchrisinger) [inline addon/components/search-dropdown/template.hbs]: I'm not seeing this behavior. When I make the change locally the "Search' placeholder text works as expected and I get the correct value in `updateSearch`. Can you please double check this?

Comment (pattisdr) [inline addon/components/search-dropdown/template.hbs]: I must be misunderstanding what you want here. 

I did this:
```         
 <input id="searchPageFullBar" value=target.value onchange={{action "updateQuery"}} name="search-placeholder" type="text" class="osf-search-input form-control" placeholder="Search">
````
which puts placeholder as target.value.  How do I reference this value to pass it into updateQuery?

**Comment by pattisdr** [comment]
Requested changes made.

**Pull request merged**